modules/add-contextual-data/tests/test_context_info_db.c

```diff
@@ -489,3 +489,52 @@ Test(add_contextual_data, test_ignore_case_off)
   context_info_db_free(db);
   contextual_data_record_scanner_free(scanner);
 }
+
+Test(add_contextual_data, test_selected_nvpairs_when_ignore_case_on)
+{
+  gchar csv_content[] = "selector,name1,value1\n"
+                        "SeLeCtOr,name2,value2\n"
+                        "sElEcToR,name3,value3\n"
+                        "selektor,name4,value4";
```

Comment:
I would write something very different than `selektor`. It would suggest some similarity in behaviour with the three lines above, but the case is the opposite.

Reply:
@furiel : well, I can modify it :)
Kira is still failing (reload_and_modified_or_deleted testcase failed... tomorrow I'll check the test file).